janitor/functions.py

```diff
@@ -2570,3 +2570,35 @@ def find_replace(df: pd.DataFrame, column: str, mapper: dict):
     """
     df[column] = df[column].apply(lambda x: mapper.get(x, x))
     return df
+
+
+@pf.register_dataframe_method
+def update_where(df: pd.DataFrame, conditions: None, target_col: None, target_val: None):
+    """
+    Add multiple conditions to update a column in the dataframe.
+    Example usage:
+    
+    .. code-block:: python
+        
+        df = (
+            pd.DataFrame(...)
```

Comment:
@jcvall, apologies for the re-comment on the example, but I think lines 2583 and 2584 here can be omitted!